.golangci.yml

```diff
@@ -0,0 +1,35 @@
+service:
+  golangci-lint-version: 1.21.0
+
+run:
+  timeout: 5m
+
+linters-settings:
+  gofmt:
+    simplify: true
+  govet:
+    check-shadowing: true
+    enable-all: true
+
+linters:
+  disable-all: true
+  enable:
+    - deadcode
+    - gofmt
+    - gosimple
+    - govet
+    - ineffassign
+    - structcheck
+    - unconvert
+    - unused
+    - varcheck
+    # TODO: enable this later
+    # - errcheck
```

Comment:
Especially for https://github.com/mattermost/mattermost-plugin-starter-template I would push for include even more linters. https://github.com/mattermost/mattermost-marketplace/blob/master/.golangci.yml might be a good source for inspiration.

But this request in not blocking the PR from going in.

Reply:
Sounds good.